assets/javascripts/modules/ajaxFormSubmit.js

```diff
@@ -128,15 +135,21 @@ var ajaxFormSubmit = {
   },
 
   serializeForAjax: function(formScope) {
-    var ret = ['isajax=true'];
+    var result = ['isajax=true'];
```

Comment:
Yes as it doesn't work for radio input type currently and is required by the consuming FE service.